main.js

```diff
@@ -33,4 +33,38 @@ const arr3 = [2, 5, 8, 15, 0, 6, 20, 3];
 		if (arr3[i] % 2 === 0) {
 			console.log(arr3[i]);
 		}
-	}	
\ No newline at end of file
+	}	
+
+	//lesson-3
+
+let palindromeWord = 'kok';
+const newPal = palindromeWord.split('').reverse().join('');
+function palidrome() {
+	(newPal === palindromeWord) ? console.log(true) : console.log(false); 
+}
+palidrome();
```

Comment:
function should return true or false, don't use console.log() for that, just `return newPal === palindromeWord`